08_files_lab/01.py

```diff
@@ -1,6 +1,19 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+import argparse 

```

Comment:
You accidentally submitted the solution to the wrong exercise here. Should take 2 file named and append the second file's contents to the end of the first one